### ISSUE TITLE
What version of Angular is supported by the storybook?

### ISSUE THREAD

**Issue by tomaszs** [storybookjs/storybook#10191]
Hello,

I was browsing through the release readme, and can not find information about what Angular versions are supported and by what versions. How can I find this information out?

**Comment by shilman** [comment]
cc @kroeder @gaetanmaisse

**Comment by kroeder** [comment]
There's no overview. But the Storybook pipeline currently tests Angular 6+ (see https://github.com/storybookjs/storybook/tree/next/lib/cli/test/fixtures)

There's also no official rule (afaik) or guarantee what Angular versions are compatible with what Storybook versions but we always work on making the latest Storybook version compatible with the latest Angular version.

Also, when there is an issue that only affects a newer version of Angular and the fix itself would break older Angular versions there would be most likely a check that ensures the fix will only be applied if you run the bug-causing Angular version.

To sum up: the tests for the last 4 major releases are green. Therefore, the latest Storybook version should work with Angular 6. 🤞 

@tomaszs I hope this answers your question? If not, let me know!

**Comment by tomaszs** [comment]
Thanks @kroeder . Yes, it answers my question. So it is tested with the latest version of Angular, and 3 earlier releases. So now when the latest version of Angular is 9, it is tested with 9,8,7 and 6. Keep up the good work!
